Corona-Warn-App/src/deviceForTesters/res/layout/fragment_test_ccl.xml

```diff
@@ -110,40 +110,5 @@
 
         </LinearLayout>
 
-        <LinearLayout
```

Comment:
Why did you to remove this from the test menu? I can see some benefits of having a simple way of opening up the consent / success screens when we need to do changes in the future...

Reply:
For this you can use screenshots testing , test menu should only include testing helpers that will stay in the app forever. it is ok to have this in the menu during development

Reply:
Okay. Makes sense. Thanks!